### PR TITLE
MWPW-139416

### PR DESCRIPTION
**Pull request by Blainegunn**
Moving `CODEOWNERS` so it the changes can be ignored and not passed to `main`
Auto Raise PR workflow to help managing stage process.
Resolves: [MWPW-139416](https://jira.corp.adobe.com/browse/MWPW-139416)

Linked to: https://github.com/adobecom/milo/pull/1949

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://MWPW-139416--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

File: .github/workflows/pull-request.yml
Comment (mokimo) [inline .github/workflows/pull-request.yml:13]: Can we clone this action? Seems like the safer thing to do & we will likely need to adapt it anyway

**Review by mokimo — COMMENTED** [review]
I don't understand on how this would work with [this PR](https://github.com/adobecom/milo/pull/1949)

Once you merge stage into main, you would always include the `.github/CODEOWNERS` file and run into merge conflicts

**Review by mokimo — CHANGES_REQUESTED** [review]
Given the pending merge-conflict question & security risk assosciated with the action, this will need an iteration.

**Comment by Blainegunn** [comment]
@mokimo 
I forgot to include the `.gitignore` update. It's here now and will insure there's no conflicts. Already tested on other projects.

As far as the security concerns go. 
>GitHub Actions can only read a secret if you explicitly include the secret in a workflow.

from, [Using secrets in GitHub Actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions)

File: .gitignore
Comment (mokimo) [inline .gitignore:9]: So in this case, what is the model of updating the codeowners, since changes are ignored? 🤔

Comment (mokimo) [reply .gitignore:9]: I am fairly sure tracked files are no longer ignored and this will lead to merge conflicts, I don’t mind testing this out though - if u previously tested this on a fork and didn’t encounter any issues.

File: .github/workflows/pull-request.yml
Comment (mokimo) [inline .github/workflows/pull-request.yml:17]: This will fail certain checks, such as the PSI tests.
Can we actually get a meaningful PR template that also includes links to all the relevant T1 stage pages & a publicly accessible milo one for PSI? It would always be main—milo—adobecom & stage—milo—adobecom

File: .gitignore
Comment (mokimo) [reply .gitignore:9]: Tested this, this leads to merge conflicts. https://github.com/mokimo/milo/pull/3

Comment (Blainegunn) [reply .gitignore:9]: @mokimo the ignore [file](https://github.com/mokimo/milo/blob/main/.gitignore) isn't merged yet

Comment (mokimo) [reply .gitignore:9]: It is on stage, but even merging it on main is irrelevant - I've included it on main as well and still has the same conflicts.

File: .github/workflows/pull-request.yml
Comment (mokimo) [reply .github/workflows/pull-request.yml:17]: I would not say the PR description does not matter. Anyone can look into those PRs, the descriptions serve an important role from a git/history perspective - also the SOTs & also devs should have a few links handy to click and go through each project.

Comment (mokimo) [reply .github/workflows/pull-request.yml:17]: An engineer that looks into stage->main sync, should also always be able to know what changes are about to go live and block them if any issues are encountered. We want to improve the process, not shift every responsibility away from an engineer to the SOTs

Comment (honstar) [reply .github/workflows/pull-request.yml:17]: I agree with @mokimo here: having meaningful PR descriptions will definitely be helpful for reviewers and approvers to understand the context of changes planned to be merged into `main`.

Since checks are run regardless of the branch, we should make sure they are green no matter what.

File: .gitignore
Comment (honstar) [reply .gitignore:9]: Looks like this setup needs a bit of rework to make this work.

**Pull request closed without merging**
